recipe/conda_build_config.yaml

```diff
@@ -3,3 +3,9 @@
 # available on 10.13
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.13"                # [osx and x86_64]
+
+# 2022/02/06 ngam
+# grpc_cpp 1.43 forces bazel 4.0.0
```

Comment:
This is because we haven't built bazel correctly for grpc 1.43 not sure why the migrator thinks it is done.

Reply:
Can you please reference 
https://github.com/conda-forge/bazel-feedstock/pull/90

In the comment.

Reply:
(added)